e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java

```diff
@@ -116,6 +124,101 @@ public void searchResultCancel() {
                 .close();
     }
 
+    @Test
+    @TestCase(value = {"EPMCMBIBPC-2658"})
+    public void searchForPipeline() {
+        library()
+                .cd(folder)
+                .clickOnDraftVersion(pipeline)
+                .configurationTab()
+                .editConfiguration(defaultProfile, profile ->
+                        profile
+                                .expandTab(EXEC_ENVIRONMENT)
+                                .expandTab(ADVANCED_PANEL)
+                                .clear(NAME).setValue(NAME,configurationName)
+                                .clear(DISK).setValue(DISK,configurationDisk)
+                                .setTypeValue(configurationNodeType)
+                                .sleep(2, SECONDS)
+                                .click(SAVE)
+                                .sleep(2, SECONDS)
+                );
+        home().sleep(2, MINUTES);
+        search()
+                .search(pipeline)
+                .enter()
+                .sleep(2, SECONDS)
+                .ensureAll(GlobalSearchAO.disable, FOLDERS, RUNS, TOOLS, DATA, ISSUES)
+                .ensureAll(enabled, PIPELINES)
+                .ensure(PIPELINES, hasText("3 PIPELINES"))
+                .validateCountSearchResults(3)
+                .hover(SEARCH_RESULT)
+                .openSearchResultItem(pipeline)
+                .ensure(TITLE, hasText(pipeline))
+                .ensure(HIGHLIGHTS, hasText("Found in name"))
+                .ensure(INFO, hasText("draft-"))
+                .parent()
+                .openSearchResultItem("docs/README.md")
+                .ensure(TITLE, hasText("README.md"))
+                .ensure(HIGHLIGHTS, hasText("Found in pipelineName"))
+                .ensure(PREVIEW, hasText("Job definition"))
+                .parent()
+                .openSearchResultItem("config.json")
```

Comment:
"config.json" is presented 4 times in the method. Let's introduce this to variable

Reply:
Done